op-deployer/pkg/deployer/upgrade/flags.go

```diff
@@ -0,0 +1,34 @@
+package upgrade
+
+import (
+	"github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer"
+	v200 "github.com/ethereum-optimism/optimism/op-deployer/pkg/deployer/upgrade/v200"
```

Comment:
Can the alias be removed here? It seems like its just aliasing to the same name that would be used anyways